src/dom4.js

```diff
@@ -721,6 +721,9 @@
         return find(this, querySelectorAll, css);
       };
       function find(node, method, css) {
+        if (node.type != document.ELEMENT_NODE) {
```

Comment:
makes sense, but do you mind dropping the square brackets to be slightly more consistent with the rest of the code?

Reply:
Sure!

Reply:
Done!